nft-demo/src/App.js

```diff
@@ -20,23 +24,35 @@ function NFTCollection(props) {
     fetch();
   })
   return (
-      <div>
-      <h1>Collection, {collectionId}</h1>
-      <table>
-        {tokenInfo?.map(({tokenId, tokenDetails, owner}) => (
-            <tr key={tokenId.toString()}>
-              <td>
-                {tokenId.toString()}
-              </td>
-              <td >
-                {tokenDetails.toString()}
-              </td>
-              <td>
-                {owner.toString()}
-              </td>
-            </tr>
-        ))}
-      </table>
+      <div className='nft_container'>
+        {tokenInfo?.map(({tokenId, tokenDetails, owner}) => {
+          tokenId = tokenId.toString()
+          tokenDetails = JSON.parse(tokenDetails.toString())
+          owner = owner.toString()
+          return (
+          <div className="flip-card">
+            <div className="flip-card-inner">
+              <div className="flip-card-front">
+                <div className={cardHovered ? 'shine' : ''}>
+                  <img
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images
Can we add this property
alt={'xxx'}

Reply:
yup sure thing